File: CS/CustomWizardExample/Wizard/Views/IChooseDataSourcePageView.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CustomWizardExample.Wizard.Views {
    public interface IChoosePersonsDataSourcePageView {
        PersonsDataSourceType DataSourceType { get; set; }
        bool ShowWarning();
    }
}

File: VB/CustomWizardExample/Wizard/Views/IChooseDataSourcePageView.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks

Namespace CustomWizardExample.Wizard.Views
	Public Interface IChoosePersonsDataSourcePageView
		Property DataSourceType() As PersonsDataSourceType
		Function ShowWarning() As Boolean
	End Interface
End Namespace
